.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,49 @@
+name: "\U0001F41E Bug report"
+description: Create a report to help us improve
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **Before You Start...**
+
+        This form is only for submitting bug reports. If you have a usage question
+        or are unsure if this is really a bug, make sure to:
+
+        - You have a local rust environment
```

Comment:
Rust enviroment is not required. The only requirement is that you must have GLIBC >= 2.32 when using Farm on linux